app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -931,6 +931,20 @@ table#user-badges {
   }
 }
 
+.simple-list-input {
+  display: flex;
+
+  input {
+    margin: 0;
+    box-sizing: border-box;
+    flex: 1 0 0px;
```

Comment:
why 0px and not 0 ?

Reply:
I copied the styles from the value-list SCSS, but found that this was an IE11 fix: 

https://github.com/discourse/discourse/commit/ba5e17a7a98af178c75d00d7be817c471fc88911 

I know we don't support IE11 anymore, but I would keep this as 0px for consistency.

Reply:
👍